deploy/runtools/deploy_managers/vitis.py

```diff
@@ -0,0 +1,327 @@
+""" Run Farm management. """
+
+from __future__ import annotations
+
+import re
+import logging
+import abc
+from fabric.api import prefix, local, run, env, cd, warn_only, put, settings, hide # type: ignore
+from fabric.contrib.project import rsync_project # type: ignore
+from os.path import join as pjoin
+
+from util.streamlogger import StreamLogger
+from runtools.run_farm_deploy_managers import InstanceDeployManager
+
+from typing import List, Dict, TYPE_CHECKING
+if TYPE_CHECKING:
+    from runtools.run_farm import Inst
+
+rootLogger = logging.getLogger()
+
+class VitisInstanceDeployManager(InstanceDeployManager):
+    """ This class manages a Vitis-enabled instance """
+    def __init__(self, parent_node: Inst) -> None:
+        super().__init__(parent_node)
+
+    def instance_logger(self, logstr: str) -> None:
+        rootLogger.info("""[{}] """.format(env.host_string) + logstr)
+
+    def clear_fpgas(self) -> None:
```

Comment:
This scares me. Seems broken in multi-tenant environments.

Can we just skip this for vitis for now?

Reply:
While I agree that this will break on multi-tenant environments, I was just going to keep this for this release. 

I think we want to clear the FPGA in use (at minimum) no?

IIRC, `validate` does "flash" and clear the old bitstream from the FPGA (in addition to running tests). However, the reset command seems better.